src/components/sheet/sheet-add-gmail.vue

```diff
@@ -36,10 +38,14 @@ const submit = async () => {
   error.value = "";
   try {
     await capture.login(form.value);
+    await store.gmail.set(BulletState.SYNC)
     form.value = { username: "", password: "", type: GMAIL };
-    capture.scan().catch((error) => console.error(error.toString()));
+    capture.scan().catch((error) => console.error(error.toString())).finally(async ()=>
+      await store.gmail.set(BulletState.P100)
+    );
     emit("back");
   } catch (err: any) {
+    await store.gmail.set(BulletState.NULL)
```

Comment:
If there is an error, it should be set to BulletState.Error not Null

Reply:
Changed